_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,286 @@
+---
+layout: post
+title: Highlights from the DBT Community's Coalesce Conference 
+date: 2021-01-25
+tags: data dbt elt etl transformation
+author: Paul Symons
+image: img/blog/coalesce2020/banner.jpg
+---
```

Comment:
Hey Paul, you may want to include a `description` tag here. When you paste the article into slack and other social media sites, it should pull out the description field as the text part.

Eg 
![image](https://user-images.githubusercontent.com/8016590/105685428-03fed280-5f31-11eb-8efa-1bcb0785a771.png)

Reply:
Thanks @jgunnink - I didn't realise that tag was available. I've added it to this article and also a few others I've written last year. Cheers!